src/Azs.Backup.Admin/Azs.Backup.Admin.psd1

```diff
@@ -0,0 +1,25 @@
+@{
+  GUID = '6b8410e2-074e-44c1-81f1-410accf696a5'
+  RootModule = 'Azs.Backup.Admin.psm1'
+  ModuleVersion = '0.9.0'
```

Comment:
module version needs update

Reply:
Hi @bganapa, this is one of the files that were generated. Should I exclude all of them from the PR?

Reply:
Ah didnt realize that, Can you pls make a PR to remove all the generated files?

Reply:
Hi @bganapa, thanks for confirming. Updated the PR to remove generated files.